source/authentication/custom-jwt.txt

```diff
@@ -9,56 +9,87 @@ Custom JWT Authentication
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 3
    :class: singlecol
 
 .. _custom-jwt-authentication-overview:
 
-Overview
---------
-
 The Custom JWT authentication provider allows users to log in with an
```

Comment:
Should we spell out JWT on first mention? I see we spell it out a few lines down, but it probably makes more sense to move that up.
```suggestion
The Custom JSON Web Token (JWT) authentication provider allows users to log in with an
```

Reply:
In this case, since it first appears as the name of one of our Authentication Services, I think not. Generally I agree totally.